apps/www/components/layout/user-account-nav.tsx

```diff
@@ -1,101 +1,18 @@
-"use client";
+import { SignedIn, SignedOut, SignInButton, UserButton } from "@clerk/nextjs";
 
-import Link from "next/link";
-import { useRouter } from "next/navigation";
-import { useClerk } from "@clerk/nextjs";
-import { User } from "@clerk/nextjs/dist/types/server";
-import { CreditCard, LayoutDashboard, LogOut, Settings } from "lucide-react";
-
-import {
-  DropdownMenu,
-  DropdownMenuContent,
-  DropdownMenuItem,
-  DropdownMenuSeparator,
-  DropdownMenuTrigger,
-} from "@/components/ui/dropdown-menu";
-import { UserAvatar } from "@/components/shared/user-avatar";
-
-export type NormalizedUser = {
-  name: string;
-  email: string;
-  imageUrl: string;
-};
-
-interface UserAccountNavProps extends React.HTMLAttributes<HTMLDivElement> {
-  user: NormalizedUser;
-}
-
-export function UserAccountNav({ user }: UserAccountNavProps) {
-  const { signOut } = useClerk();
-  const router = useRouter();
+import { Button } from "@/components/ui/button";
 
+export function UserAccountNav() {
   return (
```

Comment:
So we dont have the dropdown that says 
- Dashboard
- Billing
- etc
- etc

Would like to still have it :)